.circleci/config.yml

```diff
@@ -0,0 +1,69 @@
+version: 2.1
+
+executors:
+  my-executor:
+    docker:
+      - image: cimg/go:1.15.3
+    working_directory: ~/sparagn-media-service
+    environment:
+      PROJECT_NAME: sparagn-media-service
+      TERM: dumb
+      SSH_USERNAME: sparagn
+
+jobs:
+  build:
+    executor: my-executor
+    steps:
+      - checkout
+      - run:
+          name: decrypting with git-secret
+          command: |
+            sudo apt-get update
+            sudo apt-get install git-secret -y
+            echo -e $GPG_KEY_CI > ./private_key.gpg
+            gpg --import ./private_key.gpg
+            git-secret reveal
+      - run: go clean && go build
+      - store_artifacts:
+          path: $PROJECT_NAME
+          destination: wor
+      - store_artifacts:
+          path: docker/
+      - store_artifacts:
+          path: resources/
+      - store_artifacts:
+          path: .
+      - store_artifacts:
+          path: script/
+      - persist_to_workspace:
+          root: ~/sparagn-media-service
+          paths:
+            - $PROJECT_NAME
+            - docker/*
+            - script/*
+            - resources/*
```

Comment:
Good, that was the problem? It could not upload before to the bucket as not available at deployment workspace the google.json file. Well done !!